code/datums/craft/recipes/misc.dm

```diff
@@ -298,11 +305,9 @@
 		list(CRAFT_MATERIAL, 5, MATERIAL_STEEL, "time" = 0),
```

Comment:
```suggestion
		list(CRAFT_MATERIAL, 8, MATERIAL_STEEL, "time" = 0),
```
To alleviate the manipulator and hook removal. I don't think 5 more cables is enough

Reply:
What would you like for me to add instead? I can make it cost plasteel or something, but I just hate recipes that require weird items since it ensures they will almost never be actually crafted

Reply:
> weird items

Manipulators are weird items? Hooks? One is craft able, the other bought for 40 credits or printed for essentially free.

Reply:
"Weird items" as in "not something that's readily available in an emergency." You probably aren't going to be making yourself a makeshift arm if you can go and make things at public lathe/buy items. Hook is craftable yes (which I was sure it wasn't until i made this PR) but why add another crafting pre-req to this recipe? It's just a pain and no-one is gonna bother. 

Tedium isn't a balancing tool. If you want me to make it more expensive or require harding crafting steps, tell me what you want, but I think manips and hooks should stay gone.

Reply:
> why add another crafting pre-req to this recipe? It's just a pain and no-one is gonna bother.

Do you know what's the actual issue, at least for me? The timers and windups. The issue is not that I have to craft 3 items or something to make one, it's that I have to watch bars turn blue over and over and over and and.

That's the issue. I don't want recipes to be dumbed down, the makeshift arm needs a hook because is it's a goddamn hook arm. Every step of a recipe should be instant. Better yet, someone proposed to upgrade crafting from item slotting, to something like NT building: have items withing 1 tile of you and craft in a single step. That's a better idea than simplifying all recipes, though I agree some needed a do over.

Reply:
If you want to keep the hook craft you need to to lower its price instead because...
![изображение](https://user-images.githubusercontent.com/57810301/210157758-da8bb95f-1ed7-447b-87f4-0b7bfd5537cf.png)

Reply:
The hook additionally made it more expensive than other prosthetics since they don't cost plasteel. I don't see why a makeshift piece of shit would use advanced materials when good prosthetics wouldn't. Digging through menus to find other per-requisite items is annoying, and I see no reason why the having the crafting recipe not be separate "dumbs it down." You're still making the hook, just in the same recipe. Even the recycle value of the makeshift arm had no plasteel.